kubernetes/client.go

```diff
@@ -136,7 +157,14 @@ func LabelsMatch(serviceLabels, filterLabels map[string]string) bool {
 	return labelMatch
 }
 
-func GetLabeledListOptions(labelSelector string) *meta_v1.ListOptions {
-	return &meta_v1.ListOptions{
-		LabelSelector: labelSelector}
+func selectorToString(selector map[string]string) string {
```

Comment:
How do you much this? Maybe I am being so picky, but when I discovered I though was useful (a lil bit ruby-ish)
```go
querySelector := make([]string, 0, len(labelSelector))
for label, name := range labelSelector {
	querySelector = append(querySelector, label+"="+name)
}
return strings.Join(querySelector, ",")
```